benches/writebatch/bench_writebatch.rs

```diff
@@ -0,0 +1,131 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::time::*;
+use test::Bencher;
+use tempdir::TempDir;
+use rocksdb::{DB, WriteBatch, Writable};
+
+fn bench_writebatch_impl(round: usize, batch_keys: usize) {
```

Comment:
remove prefix `bench_` since it's only a helper for bench test, not a bench function